docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -123,9 +128,128 @@ outbreaks due to lack of vaccination in small communities.
 Data Description
 ----------------
 
-.. todo::
+.. list-table:: Definitions
+   :widths: 25 50
+   :header-rows: 1
+
+   * - State
+     - Definition
+   * - S
+     - Susceptible to measles
+   * - I
+     - Infected with measles
+   * - R
+     - Recovered from measles
+
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - 1-prevalence_c341
+     - 
+   * - S
+     - excess mortality |br| rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_c341
+     - 
+   * - I
+     - excess mortality |br| rate
+     - :math:`\frac{\text{deaths_c341}}{\text{population * prevalence_c341}}`
+     - 
+   * - I
+     - disability weights
+     - disability_weight_s117* |br| prevalence_s117+ |br| disability_weight_s118* |br| prevalence_s118
+     - GBD assumes 50% of measles |br| cases as severe and other 50% |br| as moderate |br| [GBD-2017-YLD-Capstone-Appendix-1]_.
+   * - R
+     - prevalence
+     - 0
+     - Clearly room for improvement. |br| This is done to simplify the model |br| as the focus is on LSFF but not |br| on measles.
+   * - R
+     - excess mortality |br| rate
+     - 0
+     - 
+   * - R
+     - disabilty weights
+     - 0
+     - 
+   * - ALL
+     - cause specific |br| mortality rate
+     - :math:`\frac{\text{deaths_c341}}{\text{population}}`
+     - 
+
+
+.. list-table:: Transition Data
+   :widths: 10 10 10 30 30
+   :header-rows: 1
+   
+   * - Transition
+     - Source 
+     - Sink 
+     - Value
+     - Notes
+   * - i
+     - S
+     - I
+     - incidence_rate_c341
```

Comment:
As per @beatrixkh 's comment in https://github.com/ihmeuw/vivarium_research/pull/60, we should change this to `incidence_rate_c341 / (1 - prevalence_c341)` (or `incidence_c341 / (1 - prevalence_c341)` if we're leaving "rate" implicit in our variable names...), since everything in this table should be numbers we can directly pull from GBD.